### ISSUE TITLE
Update to Julia 0.4

### ISSUE THREAD

**Issue by mauro3** [mauro3/IVPTestSuite.jl#3]
Get rid of deprecations, re-run benchmark, update results.

@obiajulu: I had some old, unpushed commits lying around on my laptop, which I now pushed.  I also merged your `dev1` branch into that.  With that the two examples run for me but not all the tests.  How about you take it from here, i.e. make a new branch (e.g. `ob/j04-fix`) and continue?

The problem with the tests: https://travis-ci.org/mauro3/IVPTestSuite.jl/jobs/132831195

**Comment by mauro3** [comment]
On the `ob/j04-fix` branch and ODE.jl `dev` branch, I still get the error the travis log also shows:

```
threebody
Solver ODE.ode45_dp (adaptive, ODE, non-stiff, explicit)
ERROR: LoadError: LoadError: test failed: 0.2978325302972853 < 0.001
 in expression: abs(re.scd - scd) < 0.001
 in error at ./error.jl:21
 in default_handler at test.jl:28
 in do_test at test.jl:53
 [inlined code] from /home/mauro/.julia/v0.4/IVPTestSuite/test/testcases.jl:34
 in anonymous at no file:0
 in include at ./boot.jl:261
 in include_from_node1 at ./loading.jl:320
 in include at ./boot.jl:261
 in include_from_node1 at ./loading.jl:320
 in process_options at ./client.jl:280
 in _start at ./client.jl:378
```

@obiajulu do you see that too?  Have you investigated this?

**Comment by mauro3** [comment]
Also, for now you should probably remove `using Compat` (and `@compat`) as, I think, we should only target 0.4.

**Comment by obiajulu** [comment]
Travis builds are passing for Julia v0.4.

**Comment by mauro3** [comment]
Well, we need to merge your branch first to close this.

**Comment by obiajulu** [comment]
ob/j04-fix has been successfully merged into master with Travis build for v0.4 is passing.
